app/services/induction/set_school_cohort_appropriate_body.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class Induction::SetSchoolCohortAppropriateBody < BaseService
+  def call
+    set_appropriate_body
+    school_cohort.save!
+  end
+
+private
+
+  attr_reader :school_cohort, :appropriate_body_id, :appropriate_body_appointed
+
+  def initialize(school_cohort:, appropriate_body_id:, appropriate_body_appointed:)
+    @school_cohort = school_cohort
+    @appropriate_body_id = appropriate_body_id
+    @appropriate_body_appointed = appropriate_body_appointed
+  end
+
+  def set_appropriate_body
+    if !appropriate_body_appointed
+      school_cohort.appropriate_body_unknown = true
+      school_cohort.appropriate_body = nil
```

Comment:
Could we not just use the presence of the `appropriate_body` to indicate whether the appropriate body is known or not?  Or is there something subtle I'm missing?

Reply:
@tonyheadford we need to show the "Appropriate body" detail line in the dashboard only for the school cohorts that went through the appropriate body selection. 

To recap:
* Existing school cohorts: don't show the appropriate body detail
* School cohorts with appropriate body unknown: show a detail line with an empty value and an "Add" action
* School cohorts with an appropriate body selected: show the name of the appropriate body with no action

In order to achieve that we need to differentiate the school cohorts where an appropriate body was never selected and the ones where the appropriate body is unknown.

Reply:
Gotcha, thought there might be something I was missing 👍